src/java.base/share/classes/java/io/FilePermission.java

```diff
@@ -181,8 +181,7 @@ public final class FilePermission extends Permission implements Serializable {
     private static final java.nio.file.FileSystem builtInFS =
         DefaultFileSystemProvider.theFileSystem();
 
-    private static final Path here = builtInFS.getPath(
-            GetPropertyAction.privilegedGetProperty("user.dir"));
+    private static final Path here = builtInFS.getPath(System.getProperty("user.dir"));
```

Comment:
Could be updated to use StaticProperty:
```suggestion
    private static final Path here = builtInFS.getPath(jdk.internal.util.StaticProperty.userDir());
```

Reply:
So changed in c7ba335.